src/components/themed/ModalParts.tsx

```diff
@@ -14,7 +21,7 @@ interface ModalTitleProps {
   icon?: React.ReactNode
 }
 interface ModalFooterProps {
-  onPress: () => void
+  onPress?: () => void
```

Comment:
This prop should be mandatory. If you don't pass it, the "X" will do nothing, and that's a bad UX. Please remove the `?`